include/fmt/chrono.h

```diff
@@ -118,6 +145,12 @@ FMT_CONSTEXPR const Char* parse_chrono_format(const Char* begin,
     case 'p':
       handler.on_am_pm();
       break;
+    case 'Q':
+      handler.on_value();
```

Comment:
`on_value` is a bit too generic. What kind of value is allowed here, time?

Reply:
Renamed to clarify the intent.